src/tests/system/tests/test_authentication.py

```diff
@@ -367,3 +369,80 @@ def test_authentication__user_login_with_modified_PAM_stack_provider_is_offline(
 
     finally:
         client.host.conn.exec(["authselect", "backup-restore", "mybackup"])
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.topology(KnownTopology.Samba)
+@pytest.mark.topology(KnownTopology.AD)
+def test_disable_an2ln(client: Client, provider: GenericProvider):
+    """
+    :title: Check localauth plugin config file (IPA/AD version)
+    :setup:
+        1. Create user
+    :steps:
+        1. Login as user
+        2. Run klist
+        3. Read localauth plugin config file
+    :expectedresults:
+        1. User can log in
+        2. Kerberos TGT is available
+        3. localauth plugin config file is present and has expected content
+    :customerscenario: False
+    """
+    provider.user("tuser").add()
+
+    pattern = (
+        r"\[plugins\]\n localauth = {\n  disable = an2ln\n"
+        "  module = sssd:/.*/sssd/modules/sssd_krb5_localauth_plugin.so\n }"
```

Comment:
Imo, to fit test purpose better, it's enough to make sure `disable = an2ln` is there and don't specify anything else.
This would make test more robust.

Reply:
Hi,

`disable = an2ln` has to be in the proper section and sub-section and since we don't have any other tests for this file I'd prefer to keep it this way.

bye,
Sumit